### ISSUE TITLE
Error opening Excalidraw

### ISSUE THREAD

**Issue by topher200** [excalidraw/excalidraw#7355]
![image](https://github.com/excalidraw/excalidraw/assets/206988/4cb2db12-ee41-4361-a413-26ea88b56b12)

Chrome: Version 117.0.5938.88 (Official Build) (64-bit)
Ubuntu 23.04

**Comment by dwelle** [comment]
Hi, this is on excalidraw.com?

**Comment by topher200** [comment]
Yes, it works on excalidraw.com but fails on app.excalidraw.com (when I go into one of my Plus drawings)

**Comment by dwelle** [comment]
It looks like it could be a VPN/firewall issue. I'm assuming loading user preferences to set the flag won't work, but you can pop into the dev console (`Opt-Cmd-I` on mac or `Ctrl-Shift-I` on Win) and paste this:

```
localStorage.setItem('FIRESTORE_LONG_POLLING', 'true')
```

Then reload the tab.

Let me know if it helps!

**Comment by topher200** [comment]
I do use a VPN for work, but I disabled it and I still see the issue.

I tried the console command and a hard refresh but the problem persists.

![image](https://github.com/excalidraw/excalidraw/assets/206988/fa7153b1-ea8f-4126-9195-5e70af2110f8)

**Comment by dwelle** [comment]
Hmm. This thing concerns me:

![image](https://github.com/excalidraw/excalidraw/assets/5153846/fccb0d9f-2a52-49bf-9b10-f9bfd983dae0)

Possible your ISP / firewall somewhere down the line is blocking the QUIC protocol.

You can try disabling it in the chrome flags: `chrome://flags/` (paste in the address bar)

![image](https://github.com/excalidraw/excalidraw/assets/5153846/c56b5134-b3eb-42be-9d28-b43153c6d53c)

Not quite sure that'll help though.

**Comment by topher200** [comment]
I reopened today and see errors which aren't the QUIC error

![image](https://github.com/excalidraw/excalidraw/assets/206988/196b067d-3b1a-49e7-927c-1a59e5f092ad)
